nixos/modules/config/users-groups.nix

```diff
@@ -53,37 +53,80 @@ let
         "*"    # password unset
       ]);
 
+  overrideOrderMutable = ''
+    {option}`initialHashedPassword` -> {option}`initialPassword` ->
+    {option}`hashedPassword` -> {option}`password`
```

Comment:
Isn't there an arrow missing? Same for the ordering below.

Reply:
Thanks for catching that!